### ISSUE TITLE
Possible Error in the Statistics for April 2024

### ISSUE THREAD

**Issue by Jacob-Zhou** [acl-org/arr-health#23]
Hi,

I noticed that in the statistics for April 2024, the total number of submissions is 881. However, the submissions for scores of 2.0, 2.5, and 3.0 are each over 600. Their combined total is much higher than 881, which seems incorrect. 

https://github.com/acl-org/arr-health/blob/fb36cb2d07a4fd207a1bd11f0b1a3a97650869a1/iterations/2024/april/conference_stats.json#L8-L11

https://github.com/acl-org/arr-health/blob/fb36cb2d07a4fd207a1bd11f0b1a3a97650869a1/iterations/2024/april/review_scores.csv?plain=1#L8-L11

**Comment by jkkummerfeld** [comment]
Yes, the current stats have several issues that we are working on resolving (one of the tech team members is actively working on this).

**Comment by jkkummerfeld** [comment]
Also note that each paper receives multiple reviews, so those numbers will not add up to 881.

**Comment by Jacob-Zhou** [comment]
Thank you for the reply and for all your hard work maintaining the statistics!

**Comment by Jacob-Zhou** [comment]
![image](https://github.com/user-attachments/assets/72dd8ac0-f67d-4525-a495-ea28f52de808)

Hi, sorry to bother you again, but I'm still a bit confused.

Does this mean that the current distribution shows individual review scores rather than the average scores per paper?
However, I noticed that the label on the x-axis still reads `Avg. Review Scores per Paper`, which seems inconsistent.

**Comment by cryptexcode** [comment]
Updated, thanks for helping to make the site better :)
